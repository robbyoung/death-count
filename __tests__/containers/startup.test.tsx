import React from 'react';
import renderer from 'react-test-renderer';
import Startup from '../../app/containers/startup';

const fakeNavigation = {
    dispatch: () => undefined,
}

jest.mock('react-navigation', () => ({
    StackActions: {
        reset: jest.fn(),
    },
    NavigationActions: {
        navigate: jest.fn(),
    }
}));

describe('Startup Container', () => {
    it('Renders with nothing in the state', async () => {
        const component = renderer.create(<Startup navigation={fakeNavigation as any} />);
        expect(component.toJSON()).toMatchSnapshot();
    });
});
