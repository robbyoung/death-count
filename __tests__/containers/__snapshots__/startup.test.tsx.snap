// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Startup Container Renders with nothing in the state 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#592a29",
      "height": "100%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#ffffff",
        "fontSize": 22,
        "fontWeight": "bold",
        "margin": 10,
      }
    }
  >
    What game are you playing?
  </Text>
  <View
    style={
      Object {
        "alignContent": "center",
        "borderRadius": 5,
        "flexDirection": "row",
        "justifyContent": "center",
        "padding": 5,
        "width": "100%",
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      onChangeText={[Function]}
      placeholder=""
      rejectResponderTermination={true}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderBottomLeftRadius": 10,
          "borderTopLeftRadius": 10,
          "flexGrow": 100,
          "fontSize": 22,
          "height": 50,
          "paddingLeft": 10,
        }
      }
      underlineColorAndroid="transparent"
      value=""
    />
    <View
      accessible={true}
      clickable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Object {
              "color": undefined,
            },
            Object {
              "backgroundColor": "#e84641",
              "borderBottomRightRadius": 10,
              "borderTopRightRadius": 10,
              "color": "#ffffff",
              "fontSize": 25,
              "height": 50,
              "textAlign": "center",
              "textAlignVertical": "center",
              "width": 50,
            },
            Object {
              "fontFamily": "FontAwesome5FreeSolid",
            },
          ]
        }
      >
        ï€Œ
      </Text>
    </View>
  </View>
</View>
`;
